ts/Extensions/Annotations/Popup.ts

```diff
@@ -295,6 +295,23 @@ H.Popup.prototype = {
             lang = this.lang,
             inputName = PREFIX + type + '-' + optionName;
 
+        if (optionParamList.length === 3 && optionName === '0') { // If a label refers to multiple fields (15733)
```

Comment:
Why we use `optionParamList.length === 3` ?

Reply:
Because in the case of 'Periods' (plural), which represents an array of numbers for the input fields, it appends an index. So the `optionsParamList` will look something like `['params', 'periods', '0']`. 
'Periods' seems to be the only case like this.